cfme/tests/services/test_rest_services.py

```diff
@@ -2066,3 +2083,31 @@ def test_delete_cart_from_collection(self, cart):
             tags: service
         """
         delete_resources_from_collection([cart])
+
+
+@test_requirements.rest
+@pytest.mark.tier(3)
+def test_deny_service_ordering_via_api(
+    appliance, update_advanced_settings, service_templates
+):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        caseimportance: high
+        initialEstimate: 1/10h
+        setup:
+            1. In advanced settings, update `:product:`-`:allow_api_service_ordering:` to `false`
+            2. Create a dialog, catalog and catalog item.
```

Comment:
Where is this done ? Creating dialog , catalog ?

Reply:
That is actually happening when we use `service_templates` fixture.